resources/packs/gcp/gcp.lr

```diff
@@ -47,6 +47,8 @@ gcp.project @defaults("name") {
   // GCP Compute Resources for the Project
   // compute() gcp.compute
   clusters() []gcp.project.cluster
+  // PubSub-related resources
```

Comment:
These feed docs and autocomplete, so we want to write them high-level in context
```suggestion
  // GCP GCP PubSub Resources
```

Reply:
done